lib/ostatus2/subscription.rb

```diff
@@ -44,7 +44,7 @@ def valid?(topic_url)
     # @return [Boolean]
     def verify(content, signature)
       hmac = OpenSSL::HMAC.hexdigest('sha1', @secret, content)
-      signature == "sha1=#{hmac}"
+      signature.downcase == "sha1=#{hmac}"
```

Comment:
Is it supposed to be a case-insensitive comparison? If so, maybe `#casecmp` + `#zero?`?